packages/larva-patterns/modules/vlanding-video-card/vlanding-video-card.prototype.js

```diff
@@ -1,8 +1,22 @@
 const clonedeep = require( 'lodash.clonedeep' );
 
 const c_lazy_image = clonedeep( require( './../../components/c-lazy-image/c-lazy-image.prototype' ) );
+const c_title = clonedeep( require( './../../components/c-title/c-title.prototype' ) );
+const c_span = clonedeep( require( './../../components/c-span/c-span.prototype' ) );
 
 module.exports = {
 	vlanding_video_card_permalink_url: "#",
+	vlanding_video_card_link_showcase_trigger_data_attr: '',
```

Comment:
Making a note that we might refactor this to include a string of JSON instead of the indiviudal items - e.g. `vlanding_video_card_link_showcase_data_attr` - but for now this is great!